bazel_integration_test/py/test_base.py

```diff
@@ -252,8 +252,10 @@ def _EnvMap(self, env_remove = None, env_add = None):
       java_home = TestBase.GetEnv('JAVA_HOME', '')
       if java_home:
         env['JAVA_HOME'] = java_home
+      # TODO: figure out Windows runfiles_dir
```

Comment:
Let's create an issue and replace `TODO` with the `GHXXX` where `XXX` is the issue number.

Reply:
Broke this out to a new issue (#182) and a new PR (#183).